vm_automation/esxiVm.py

```diff
@@ -513,6 +513,18 @@ def runCmdOnGuest(self, cmdAndArgList):
             retVal = False
         return retVal
 
+    def scheduleCmdOnGuest(self, cmdAndArgList, secDelay):
+        # THE POINT HERE IS THAT WHEN VMWARE TOOLS RUNS EXEs, IT DOES SO WITH VERY LIMITED PRIVS
+        # CAUSING SOME PRIV ESC ATTACKS TO FAIL.  SCHEDULING THE PAYLOAD FIXES THAT.  FYI, RUNAS 
+        # DOES NOT
+        schedTime = datetime.datetime.now() + datetime.timedelta(seconds=secDelay)
+        schedTimeStr = str(schedTime.hour) + ":" + str(schedTime.minute)
+        self.server.logMsg("SCEDULE TIME FOR EXECUTION = " + schedTimeStr)

```

Comment:
Found a bug here.  When the minutes is less than 10, it truncates leading zeros, resulting in a bad time value.  For example, it the time that it wants to launch is 12:04, this will report it as 12:4, which fails to schedule correctly.  Working on a fix.